sky/clouds/os_image/skypilot-gcp-gpu-debian-11.yaml

```diff
@@ -0,0 +1,10 @@
+resources:
+  cloud: gcp
+  accelerators: L4:1
+  use_spot: True
+  disk_size: 50
+  image_id: projects/deeplearning-platform-release/global/images/common-cu121-v20231105-debian-11-py310
+
+setup: |
+  sudo rm -rf /home/gcpuser/.config/gcloud/
```

Comment:
We may want to delete all the other clouds credentials, since a person may have multiple other clouds credentials uploaded as well. : )

Reply:
Created a ticket to address this: https://linear.app/skypilot/issue/SKY-662/remove-other-cloud-credentials-from-image